src/fhcount.c

```diff
@@ -0,0 +1,142 @@
+/**
+ *
+ * collectd - src/fhcount.c
+ * Copyright (C) 2015  Jiri Tyr
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
```

Comment:
The majority of our code is moving towards the ISC license (https://en.wikipedia.org/wiki/ISC_license). Could I convince you to use that license, too?